classes/Validate.php

```diff
@@ -543,7 +543,7 @@ public static function isDate($date)
 
     public static function isDateOrNull($date)
     {
-        if (is_null($date) || $date === '0000-00-00 00:00:00') {
+        if (is_null($date) || $date === '0000-00-00 00:00:00' || $date === '0000-00-00') {
```

Comment:
Reverse the second and third check, because it's better to check the smaller date first instead the bigger one.

Reply:
@pollarize In the case of PrestaShop, I don't think because most (maybe all since this method worked until now) of the date are in the long form.